composer.json

```diff
@@ -0,0 +1,18 @@
+{
+    "name": "php/timezonedb",
```

Comment:
There is ongoing discussion about the top level namespace `php` being used on Packagist...

Reply:
see also php/pie#65

Reply:
Updated to use `pecl/timezonedb` package name